sentry-android/src/main/java/com/joshdholtz/sentry/Sentry.java

```diff
@@ -94,6 +98,7 @@
 	private String packageName;
 	private int verifySsl;
 	private SentryEventCaptureListener captureListener;
+	private Map<String, Object> contexts;
```

Comment:
`context` is perfect! I've been making the Swift client as that is what I named it too 👌